_includes/assets/sass/base/_typography.scss

```diff
@@ -1,11 +1,17 @@
 html {
   color: $tbds-color-text-default;
   font-family: $font-family-sans;
-  font-size: 100%;
-  padding: 0 $tbds-space-6;
+  font-size: 90%;
+  padding: 0 $tbds-space-3;
+
+  @media (min-width: $tbds-breakpoint-medium) {
+    font-size: 100%;
```

Comment:
It sort of seems to me like we should have some standard font sizes in tbds. I'm not sure if there was a philosophy behind what is in tbds and what is not, but it seems a little arbitrary to me.

Anyway, since there isn't, this more more of an aside comment. 😄

Reply:
Yep I'm going to add an issue either to tads or the guide tidy add better tokens for typography.